CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ Possible log types:
 
 ### UNRELEASED
 
+- [added] Completed implementation of the `Contact` field (#6)
```

Comment:
Maybe change to *complete implementation* because *added completed implementation* sounds strange.